data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report8_household_state_genera.py

```diff
@@ -12,25 +13,35 @@
 #     name: python3
 # ---
 
-# # QA queries on new CDR_deid household AND state general
+# # QA queries on new CDR_deid household AND state generalization
 
 import urllib
 import pandas as pd
+from common import JINJA_ENV
 from utils import auth
 from gcloud.bq import BigQueryClient
 from analytics.cdr_ops.notebook_utils import execute, IMPERSONATION_SCOPES
 pd.options.display.max_rows = 120
 
 # + tags=["parameters"]
 project_id = ""
-deid_cdr = ""
-# deid_clean=""
-com_cdr = ""
-run_as = ""
-# -
+deid_cdr=""
+combine = ""
+reg_combine=''
+pipeline = ""
+deid_sand = ""
+pid_threshold=""
+run_as=""
 
-# df will have a summary in the end
-df = pd.DataFrame(columns=['query', 'result'])
+# +
+#project_id = "aou-res-curation-prod"
+#deid_cdr = "R2022q4r4_deid"
```

Comment:
This parameter should be removed, we do not want our dataset and project names in the git history.